File: API/Models/Product.cs
// API/Models/Product.cs
using System.ComponentModel.DataAnnotations;

namespace API.Models
{
    public class Product
    {
        [Key]
        public int Id { get; set; }
        public string Name { get; set; }
        public long LastModified { get; set; }
    }
}

File: MobileApp/app/src/main/java/com/example/mysyncapp/entities/Product.kt
// MobileApp/app/src/main/java/com/example/mysyncapp/entities/Product.kt
@Entity(tableName = "products")
data class Product(
    @PrimaryKey(autoGenerate = true) val id: Int = 0,
    @ColumnInfo(name = "name") val name: String,
    @ColumnInfo(name = "last_modified") val lastModified: Long
)
